### PR TITLE
GitHub-290: Removing 'recommend' as an acceptable alternative of 'we …

### PR DESCRIPTION
**Pull request by bergerhoffer**
…suggest'

#290

Preview: http://file.rdu.redhat.com/~ahoffer/2023/main-recommend.html#we-suggest

### CODE REVIEW

File: supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc [hunk 1]
```diff
@@ -34,7 +34,7 @@
 [discrete]
 [[we-suggest]]
 ==== image:images/no.png[no] we suggest (verb)
-*Description*: Do not use "we suggest". Use a more direct construction or use "recommend". For example, instead of "We suggest that you make a backup of your data disk", write "Back up your data disk", or "It is recommended that you back up your data disk".
+*Description*: Do not use "we suggest". Use a more direct construction instead. For example, instead of "We suggest that you make a backup of your data disk", write "Back up your data disk".
```
Comment (dfitzmau) [inline supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:37]: Replace the phrasal verb "back up" with archive?

Comment (mportman12) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:37]: "Back up" is fine per ISG and SSG. Any particular reason to change the wording?

Comment (dfitzmau) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:37]: A bit of minimalism. Usually, phrasal verbs should be avoided, where possible. I thought I read somewhere about these verb phrases with translations. Maybe, I misread this.

Comment (bergerhoffer) [reply supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc:37]: I'm only removing the recommended wording, not necessarily trying to reword the rest of the content. I'd probably prefer to just keep as is

**Review by dfitzmau — APPROVED** [review]

**Review by mportman12 — APPROVED** [review]
LGTM

**Review by bredamc — APPROVED** [review]
LGTM :)

**Review by sbmetz — APPROVED** [review]
LGTM

**Review by bburt-rh — APPROVED** [review]

**Pull request merged**